### ISSUE TITLE
Accessibility should be enabled by default

### ISSUE THREAD

**Issue by fcnjd** [CollaboraOnline/online#10221]
### Is your feature request related to a problem?

I need to enable accessibility since I'm screenreader dependent, but this cannot consistently be done on the Nextcloud snap, so it should be enabled by default.

### Describe the solution you'd like

Screenreader accessibility should not be a feature or an option, rather a standard. It's great that you implemented this option, but there are cases (in my case the Nextcloud snap) where modifying the coolwsd.xml is not possible, since it's only written to /tmp and as such regularly erases. A user who requires accessibility shouldn't be forced to contact an admin to do that, instead just be able to use the software as normal.

### Describe alternatives you've considered

Maybe it's possible to modify the snap so that the coolwsd.xml gets a permanent location, but I believe just making this the default makes things most easy.

**Comment by sebastiaanveld** [comment]
Maybe we should get this information from the collaboration tool COOL/CODE is part of, like Nextcloud. Could be a setting in the Office App settings, or case the user has certain themes selected in Nextcloud inherit this in COOL/CODE (like Office App "accessibility" set to "Automatic").

**Comment by timur-g** [comment]
Accessibility on server is by default false as it puts performance pressure on the server.

**Comment by sebastiaanveld** [comment]
@timur-g should we then not have an ability to setup multiple COOL/CODE configs:
- Setup A: Default (office.mydomain.ext)
- Setup B:  Accessibility enabled users (office-accessibility.mydomain.ext)
If with certain accessibility features enabled in their get redirected to setup B while any other user gets redirected to setup A.

**Comment by timur-g** [comment]
I am reopening the ticket for the last question, not sure now if the solution is to change config or run two servers.
